eval/eval_openlm_ckpt.py

```diff
@@ -1,7 +1,7 @@
 import argparse
 import json
 import time
-import sys
+import sys; sys.path.append('..')
```

Comment:
Is this necessary? If you pip install openlm, you shouldn't need this right?

Reply:
Yea good catch +1

Reply:
Continuing the conversation that's partially on Slack here. Maybe the following is useful, I've adopted this approach in the past when developing Python packages: https://setuptools.pypa.io/en/latest/userguide/development_mode.html

Reply:
Putting what I wrote on slack publicly:
-------------------------
Re the sys.path.append('..'): yes it's a bit sketchy. I hadn't installed open_lm via pip -- I generally tend to avoid assuming we've got the pip packages (especially when we're developing and making changes). We can always point pip to the directory where things are contained, but then it's tricky to make sure we're using the current under-development code.
----------------------------------
For an example output:
passing this command:
python generate.py \
--model open_lm_1b |
--checkpoint /raid/mgj528/checkpoints/open_lm/open_lm_1B/open_lm_1b.pt \
--rotary-old \
--input-text "Deep learning is defined as "
 I get this output:

Loading model into the right classes...
--------------------------------------------------
		 Model output:
--------------------------------------------------
Deep learning is defined as 
the use of neural networks as black boxes, i.e., without providing any explicit model architecture, e.g., as a generic approach for machine learning without domain expertise or a specific algorithm, or by making use of off-the-shelf libraries, to solve a variety of machine learning problems.

A popular type of machine learning method is deep learning, which has become a standard machine learning approach. In recent years, deep learning has been used to solve a variety of problems in natural language processing (NLP), such as question answering and sentiment analysis. In this context, a great deal of research has focused on developing deep learning models for NLP tasks \cite{Sutskever2014Convolutional,Hochreiter1997Long}.

Deep learning models can be trained with limited labeled data. However, to ensure that a trained model works well, additional data must be collected and labeled. This makes it difficult to adopt deep learning models in real-world applications
--------------------------------
And re: rotary old,

I just copied the exact same thing that happens with the eval/eval_openlm_ckpt.py code:
i.e.
the rotary-old kwarg gets added via open_lm.params.add_model_args and then model gets constructed via OpenLMConfig(create_params(args))

So I have no idea if this is correct, just that it's what the eval code does

Reply:
Okay awesome! The generation looks quite reasonable, let's merge this after removing `sys.path.append`. Agree with Ludwig re: `pip install -e .`, that's my usual strategy. We can add that to README if we want.

Reply:
Thanks for the PR + testing it!